### ISSUE TITLE
Add 2 more keybinds for increasing and decreasing speed controls

### ISSUE THREAD

**Issue by Blappacino** [emmVRC/emmVRC-Public#1564]
I dont want to have to navigate several layers of menus just to get to the page where i can control my speed menu. I would like to just press and hold buttons like + or - for manipulating various levels of fast and slow

**Comment by Caldoric** [comment]
I agree, having an option in the "risky features" radial/quickmenu submenu enabling adjustment of the speed value is long overdue. My opinion, though, is that it should have one of those "radial slider" puppet menus, or whatever they're called. And place it such that the actual "speed" on/off toggle is directly at the bottom of the submenu, so you can quick-flick down and call it good.
